scripts-available/CDB_Groups.sql

```diff
@@ -0,0 +1,124 @@
+-- Creates a new group
+CREATE OR REPLACE
+FUNCTION cartodb.CDB_Group_CreateGroup(group_name text)
+    RETURNS VOID AS $$
+DECLARE
+  cdb_group_role TEXT;
+BEGIN
+  -- TODO: escape group_name
+  cdb_group_role := cartodb._CDB_Group_GroupRole(group_name);
+  IF NOT EXISTS ( SELECT 1 FROM pg_roles WHERE rolname = cdb_group_role )
+  THEN
+    EXECUTE 'CREATE ROLE "' || cdb_group_role || '" NOLOGIN;';
```

Comment:
"Another restriction on parameter symbols is that they only work in SELECT, INSERT, UPDATE, and DELETE commands" :( I've used it for `INFORMATION_SCHEMA.SCHEMATA` query (the only SIUD command), much nicer.